jaxrs-api/src/main/java/jakarta/ws/rs/BadGatewayException.java

```diff
@@ -0,0 +1,105 @@
+/********************************************************************************
+ * Copyright (c) 2019 Jeyvison Nascimento
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ ********************************************************************************/
+
+package jakarta.ws.rs;
+
+import jakarta.ws.rs.core.Response;
+
+/**
+ * A runtime exception indicating that the server, while acting as a gateway or proxy,
+ * received an invalid response from the upstream server it accessed in attempting to fulfill the request.
+ *
+ * @author Jeyvison Nascimento (jeynoronha@gmail.com)
+ * @since 2.2
```

Comment:
Shouldn't this be 3.0 now?

Same for each class.

Reply:
I think it should be 3.1 - and the PR should target `3.1-SNAPSHOT` instead of `master`.

Reply:
Yes, since 3.1 and please target 3.1-SNAPSHOT